apps/cms/migrations/0004_add_projects_block.py

```diff
@@ -3,7 +3,7 @@
 
 from django.db import migrations, models
 import wagtail.wagtailcore.fields
-import apps.cms.models
+import apps.cms.blocks
```

Comment:
If I do not change this `migrate` and `makemigrations` will fail with an exception.